extractor/src/main/resources/projects-for-example.yml

```diff
@@ -0,0 +1,11 @@
+SummaryJavadocCheck:
+  Example3:
```

Comment:
SummaryJavadoc

Reply:
I am waiting for my PR in the main repo to be merged. Once that is done, we can refactor.

Reply:
@relentless-pursuit , that PR is merged. Please do update

Reply:
> @relentless-pursuit , that PR is merged. Please do update

rebased.